src/main/frontend/client/components/BerryEdit.js

```diff
@@ -0,0 +1,120 @@
+import _ from "lodash"
+import React, { useState } from "react"
+import { Redirect } from "react-router"
+import ErrorList from "./ErrorList"
+
+const BerryEdit = props => {
+  const [formPayload, setFormPayload] = useState({
+    name: "",
+    imgUrl: "",
+    description: ""
+  })
+
+  const [errors, setErrors] = useState({})
+
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+
+  const editBerry = async () => {
+  

```

Comment:
Remove this empty line and make sure to format the file as the indentation is off for certain lines here